src/common/src/array/stream_chunk.rs

```diff
@@ -88,6 +88,123 @@ impl StreamChunk {
         StreamChunk { ops, data }
     }
 
+    /// Parse a chunk from string.
+    ///
+    /// # Format
+    ///
+    /// The first line is a header indicating the column types.
+    /// The following lines indicate rows within the chunk.
+    /// Each line starts with an operation followed by values.
+    /// NULL values are represented as `.`.
+    ///
+    /// # Example
+    /// ```
+    /// use risingwave_common::array::StreamChunk;
+    /// let chunk = StreamChunk::from_str(
+    ///     "  I I I I      // type chars
+    ///     U- 2 5 . .      // '.' means NULL
+    ///     U+ 2 5 2 6 D    // 'D' means deleted in visibility
+    ///     +  . . 4 8      // ^ comments are ignored
+    ///     -  . . 3 4",
+    /// );
+    /// //  ^ operations:
+    /// //     +: Insert
+    /// //     -: Delete
+    /// //    U+: UpdateInsert
+    /// //    U-: UpdateDelete
+    ///
+    /// // type chars:
+    /// //     I: i64
+    /// //     i: i32
+    /// //     F: f64
+    /// //     f: f32
+    /// //    TS: Timestamp
+    /// ```
+    pub fn from_str(s: &str) -> Self {
```

Comment:
How about `from_pretty`?

Reply:
```
 method `from_str` can be confused for the standard trait method `std::str::FromStr::from_str`
```

Also, clippy suggest not to use `from_str`

Reply:
renamed in 04313c2